src/wallet/interfaces.cpp

```diff
@@ -111,6 +111,17 @@ WalletTxOut MakeWalletTxOut(const CWallet& wallet,
     return result;
 }
 
+WalletTxOut MakeWalletTxOut(const CWallet& wallet,
```

Comment:
`MakeWalletTxOut` is used to create entries for the GUI. Part of that is the transaction's time, retrieved with `CWalletTx::GetTxTime`. This constructor is called for each `COutput` returned by `ListCoins`. As `COutput` will no longer have access to the `CWalletTx` later in this PR, it is necessary for it to be able to have access to the time so that `WalletTxOut` can have the correct time set. That is why this change exists, and it is, in fact, related to storing time in the `COutput`.